client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -73,15 +76,47 @@ class CheckoutContainer extends React.Component {
 			upgradeIntent,
 			shouldShowCart = true,
 			clearTransaction,
-			isComingFromFrankenflow,
+			isComingFromGutenboarding,
 		} = this.props;
 
 		const TransactionData = clearTransaction ? CartData : CheckoutData;
 
 		return (
 			<>
-				{ this.renderCheckoutHeader() }
-				{ this.shouldDisplaySiteCreatedNotice() && (
+				{ this.props.isComingFromGutenboarding && (
+					<>
+						<Button
+							borderless
+							className="navigation-link back"
+							onClick={ () => window.history.go( -2 ) } // going back to signup flow and skipping '/launch' step
+						>
+							<Gridicon icon="arrow-left" size={ 18 } />
+							{ this.props.translate( 'Back' ) }
+						</Button>
+						<div className="checkout__site-created--gutenboarding">
+							<img
+								src="/calypso/images/signup/confetti.svg"
+								aria-hidden="true"
+								className="checkout__site-created-image"
+								alt=""
+							/>
+							<FormattedHeader
+								headerText={ this.props.translate(
+									'Your WordPress.com site is ready! Finish your purchase to get the most out of it.'
+								) }
+							/>
+							<div>
+								{ this.props.translate( '{{em}}%(siteSlug)s{{/em}} is up and running!', {
+									components: { em: <em /> },
+									args: { siteSlug: this.props.selectedSite.slug },
+									comment: '`siteSlug` is the WordPress.com site, e.g., testsite.wordpress.com',
+								} ) }
+							</div>
+						</div>
```

Comment:
Not reusing much except the confetti image since all these will change as @simison stated in https://github.com/Automattic/wp-calypso/pull/39447#issuecomment-590710667